build.gradle

```diff
@@ -254,13 +258,13 @@ afterEvaluate { project ->
     // debugging
     task printCompilerInput << {
       logger.info "**JAVAC INPUT FILES**"
-      compileReleaseJava.source.each {
+        compileReleaseJavaWithJavac.source.each {
```

Comment:
Please revert the indentation change, keep the same indentation level as before.